lib/notification/sendgrid/wrapper.js

```diff
@@ -0,0 +1,23 @@
+import sgMail from '@sendgrid/mail';
```

Comment:
This should be in the following folders
 lib/mongoose/js/src/config/email/sendgrid.js
 lib/mongoose/ts/src/config/email/sendgrid.ts
 lib/sequelize/js/src/config/email/sendgrid.js
 lib/sequelize/ts/src/config/email/sendgrid.ts

It should also be named sendgrid and not wrapper

Reply:
Doing this might not work because it will copy the email configuration for both SendGrid and mailgun when creating the project. When the project is created, the email config should be empty until they set it up